tests/0013-db-snapshot.toml

```diff
@@ -2,7 +2,7 @@
 default_image = "docker.io/paritypr/polkadot-debug:master"
 default_command = "polkadot"
 default_args = [ "-lparachain=debug" ]
-default_db_snapshot = "https://storage.googleapis.com/zombienet-db-snaps/chains-test.tgz"
+default_db_snapshot = "https://imbue-test-us-east-2.s3.us-east-2.amazonaws.com/chains-b37ea1fc1eba94880fdf431a769cbaad75a89199.tgz"
```

Comment:
@pepoviola @michalkucharczyk  please note the new snapshot, this needs uploading and the link replaced

Reply:
I will take care of it :+1:

Reply:
Thanks @skunert! You also need to update the version of zombienet to the latest.

Thanks!!